roles/openshift_metrics/tasks/main.yaml

```diff
@@ -43,7 +43,8 @@
   shell: >
    {{ openshift.common.client_binary }} process -f \
    /usr/share/openshift/examples/infrastructure-templates/{{ hawkular_type }}/metrics-deployer.yaml -v \
-    HAWKULAR_METRICS_HOSTNAME=hawkular-metrics.{{ openshift.master.default_subdomain }},USE_PERSISTENT_STORAGE={{ hawkular_persistence }} | \
+    HAWKULAR_METRICS_HOSTNAME=hawkular-metrics.{{ openshift.master.default_subdomain }},USE_PERSISTENT_STORAGE={{ hawkular_persistence }} \
+    METRIC_DURATION=openshift_hosted_metrics_duration,METRIC_RESOLUTION=openshift_hosted_metrics_resolution | \
```

Comment:
Should be METRIC_DURATION={{ openshift.hosted.metrics.duration }}
The tamplate doesn't support yet METRIC_RESOLUTION see [metrics-deployer.yaml](https://github.com/jdnieto/openshift-ansible/blob/ea80a6cb7fc4309e5c5c4210b2ce8504e8008c56/roles/openshift_examples/files/examples/v1.2/infrastructure-templates/enterprise/metrics-deployer.yaml)